sdk/python/tests/unit/permissions/auth/test_token_parser.py

```diff
@@ -120,3 +172,70 @@ def test_k8s_token_validation_failure(mock_jwt, mock_config):
         asyncio.run(
             token_parser.user_details_from_access_token(access_token=access_token)
         )
+
+
+@mock.patch.dict(os.environ, {"INTRA_COMMUNICATION_BASE64": "test1234"})
```

Comment:
same comments as above

Reply:
Look for my comment above it is related to this as well